presto-main/src/test/java/com/facebook/presto/sql/planner/TestSortExpressionExtractor.java

```diff
@@ -81,6 +82,61 @@ public void testGetSortExpression()
                         ComparisonExpressionType.GREATER_THAN,
                         new FunctionCall(QualifiedName.of("sin"), ImmutableList.of(new SymbolReference("b1"))),
                         new SymbolReference("p1")));
+
+        assertGetSortExpression(
+                new LogicalBinaryExpression(LogicalBinaryExpression.Type.OR,
```

Comment:
instead of creating expressions by hand you can use something like:

```
expression("b1 > p1 OR b2 <= p1")
```
where `expression` is:
```
private Expression expression(String sql) {
 return rewriteIdentifiersToSymbolReferences(new SqlParser().createExpression(sql));
}
```